Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -3914,3 +3914,13 @@
     engine: Blink
     engine_version: ""
     family: Chrome
+
+-
+  user_agent: Nokia6120c/GoBrowser/2.0.290
+  client:
+    type: browser
+    name: GoBrowser
+    version: 2.0.290
+    engine: ""
+    engine_version: ""
+    family: Unknown
```

Comment:
```suggestion
    family:
```

Reply:
in nodejs yaml parser result
`family: ` result `family: null`
I suggest making an explicitly empty string
`family: ""`